app/controllers/api/user_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::UserController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
-  skip_before_filter :verify_authenticity_token
+  skip_before_filter :verify_authenticity_token, raise: false
```

Comment:
You would think so, but this is the expected behaviour when the callback being skipped is yet to be defined (we end up in this code path): 

http://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-skip_callback